src/fides/api/models/privacy_experience.py

```diff
@@ -327,32 +563,13 @@ def unlink_experience_config(self, db: Session) -> PrivacyExperience:
 
         Note that neither the Experience Config or Experience are deleted; we're only removing
         a FK if it exists.
+        # TODO revisit this method
         """
         return self.update(  # type: ignore[return-value]
             db,
             data={"experience_config_id": None, "experience_config_history_id": None},
         )
 
-    def link_default_experience_config(self, db: Session) -> PrivacyExperience:
-        """Replace config from experience with default config
-
-        Replaces the FK only; does not delete Privacy Experiences or Privacy Experience Configs
-        """
-        default_config: Optional[
-            PrivacyExperienceConfig
-        ] = PrivacyExperienceConfig.get_default_config(
-            db, self.component  # type: ignore[arg-type]
-        )
-        return self.update(  # type: ignore[return-value]
-            db,
-            data={
-                "experience_config_id": default_config.id if default_config else None,
-                "experience_config_history_id": default_config.experience_config_history_id
-                if default_config
-                else None,
-            },
-        )
```

Comment:
We previously had a concept of a "default" Privacy Experience Config - one overlay, one privacy experience, and one tcf overlay.   This helped the backend make sure that Privacy Experiences could at least always have a default config linked to it as configuration changes were made in the background.  This concept is going away.